### PR TITLE
Audit fixes 2

### PR DESCRIPTION
**Pull request by Rashmi-278**
### Summary

**Integrate**
YungApeSquad

**Disabled** 
Dai-Prize-Pool  -  Rebranded as PoolTogether

**Add forum info** 
Pickle finance
DaoSquare
Lido
Doodles
Fei Protocol
Gitcoin
Indexed
BadgerDAO
GoodDollar
GraphProtocol

**Treasury contract address added**
Dynamic set dollar
LexPunk

**Update social links**
dYdX

**Overview additions**
Pickle Finance
Aavegotchi
Badger
Balancer
Credmark
Crypto-corgis - just in case we enable em again
DynamicSetDollar
TheGraph Protocol
InstaDAPP
Lido
OpenDAO
Pickle FInance
Premia
RARI Capital
Seen
TECommons
TokenLon
TrueFi
Yearn Finance

### CODE REVIEW

File: protocols/yungapesquad/overview.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Yung Ape DAO
```
Comment (cryptosabago) [inline protocols/yungapesquad/overview.md:1]: Should be `#`

File: protocols/yungapesquad/overview.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Yung Ape DAO is the governing body of the Yung Ape Squad that works in harmony with the project team and the community. The main responsibilities of the DAO are to track and approve expenditures, decide on project activations, fund growth on multiple verticals.
+
+## Yung Ape DAO Treasury
```
Comment (cryptosabago) [inline protocols/yungapesquad/overview.md:5]: I think that this heading and the one after should be grouped under `# Governance`

File: protocols/yungapesquad/overview.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+The spending should focus on the growth of the project and support a long-term sustainable strategy for the well-being of the DAO members.
+
+## Yung Ape Proposals
+
+Yung Ape Proposals (YAPs) are proposals that are anticipated to play a big role in the future of the project. YAPs can be perceived as alignments that drive action points, the biggest difference for Young Ape DAO will be the fact that everybody will have a voice.
+
+The YAPs process and the project’s overall development are supported by a dedicated team who will be responsible for bringing these proposals to the DAO. DAO will get to choose whether to pursue the proposed strategy or not.
+
+### Additional Resources
```
Comment (cryptosabago) [inline protocols/yungapesquad/overview.md:19]: Missing `Join our DAO` section.
And I think that this should be a `#` level heading

File: protocols/yungapesquad/index.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "yungapedao",
+  "description": "Yung Ape DAO is the governing body of the Yung Apes Squad that works in harmony with the project team and the community. The main responsibilities of the DAO are to track and approve expenditures, decide on project activations, fund growth on multiple verticals.",
+  "path": "yungapedao",
+  "type": "snapshot",
+  "isEnabled": true,
+  "treasuryAddress": ["0x76Dc0C73Dc211423bB15dAee3232eDDD52D0b725"]
+}
```
Comment (cryptosabago) [inline protocols/yungapesquad/index.json:8]: Nit: new line

File: protocols/seen/overview.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Seen.haus
+
+seen.haus is an auction house that brings the physical and digital worlds together. Seen.haus establishes itself as a trustworthy platform for creators and investors offering limited-edition high-quality pieces of digital and physical artworks.
+
+By leveraging blockchain technology, seen.haus provides immutable and publicly verifiable proof of authenticity for its listed artworks that permanently links each physical art piece with its digital representation.
+
+## Additional Links
```
Comment (cryptosabago) [inline protocols/seen/overview.md:7]: I think that this should be a `#` level heading. 
And this overview is missing a few sections such as `Join our DAO` (I see you have their links below), `governance` and potentially `overview`

File: protocols/pickle/overview.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+Pickle Finance is gas efficient
+
+Boosts for farm rewards are available if you lock PICKLE for DILL
+
+The Pickle DAO
+
+Pickle is governed by a DAO (Decentralized Autonomous Organization). Holders of “DILL”, a representation of a PICKLE token that has been time-locked on the platform for a set duration, receive voting rights to make changes to the protocol, emissions, and more. No single person is in charge. We’re all in this together.
+
+Joining the Community
```
Comment (cryptosabago) [inline protocols/pickle/overview.md:49]: `#`

File: protocols/pickle/overview.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+Our community members (AKA Pickle Privates) can be found all over the interwebs. The most active of these is without a doubt the discord server, where members discuss strategies, report bugs, or help solve each others’ problems in real-time.
+
+Additional Links
```
Comment (cryptosabago) [inline protocols/pickle/overview.md:53]: `#`

File: protocols/pickle/overview.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+Pickle aggregates and compounds yield from other protocols
+
+It saves you time and money compared to doing it yourself.
+
+Decentralized Finance (DeFi) has grown tremendously over the past year. There are many parts of DeFi, including lending platforms, liquidity protocols, stock synthetics, automated market makers, and more. Yield aggregators are another option in the Decentralized Finance space.
+
+Yield Aggregators exist for yield farmers (like you) who want to invest money and maximize profits by leveraging different DeFi protocols and strategies for elevated returns. Pickle Finance makes it easy for you to earn great compounding yields on your deposits, when you don’t have the time to compound it daily or the gas fee is too high for frequent compounding to be done. In short, Pickle Finance is always on the lookout for opportunities to generate yield on your assets for all risk tolerance levels.
+
+## How Does it Work?
```
Comment (cryptosabago) [inline protocols/pickle/overview.md:15]: Should be under `# Governance`?
Could also break down some of the sections below into `###` level subheadings: `Pickle Jars`, `Pickle Farms`, `Why use Pickle Finance` and `The Pickle DAO`

File: protocols/yungapesquad/overview.md (hunk 2 above)
Comment (Rashmi-278) [reply protocols/yungapesquad/overview.md]: fixed

File: protocols/seen/overview.md (hunk 5 above)
Comment (Rashmi-278) [reply protocols/seen/overview.md:7]: Added and formattted

File: protocols/yungapesquad/index.json (hunk 4 above)
Comment (Rashmi-278) [reply protocols/yungapesquad/index.json]: Not sure what you mean ?

**Pull request merged**